templates/posts/detail_anonymous.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+{% load static %}
+
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="{% static 'css/global.css'%}" type="text/css">
+    <meta charset="UTF-8">
+    <title>Recipes</title>
+</head>
+<body>
+<div style="margin: 0 auto;width: 500px">
+    {% if request.user.is_authenticated %}
```

Comment:
As far as I can see, there is no `request` in `context`. Is it added automatically ?

`return render(request, 'posts/detail_anonymous.html', context={'post': post})`
https://github.com/berkyaglioglu/Post-Sharing-Application/pull/7/files#diff-7f5d848739da3904921a61c635a3254bR36